### DISCUSSION TITLE
Unable to integrate supbase with my webpage

### DISCUSSION THREAD

**Discussion by Arjunheregeek** [supabase/supabase#30288] [Questions]
Can anybody help me out with establishing connection the my supa database .

**Comment by bytaesu** [comment]
[Supabase docs](https://supabase.com/docs)

Here is your friend.

If you have a specific issue and leave detailed questions, another friend will help you out instead of this one.

**Reply by Arjunheregeek** [reply]
We made a table on supbase database but we are unable to integrate it with a web page  to view the table from the web page .

**Reply by bytaesu** [reply]
Disable Row Level Security on the table and test it. If you’re working in a production environment, please look up the appropriate usage and apply it accordingly.

I can only suggest one of the common issues here; there’s no magical answer to a question like ‘It just doesn’t work.’

**Comment by silentworks** [comment]
What have you tried? what didn't work? You haven't mentioned any issues here, if you are asking how to use Supabase the docs cover this quite well and they even have quickstart docs for various frameworks too.
